packages/theme-data/src/__stories__/components/Example.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Reference from "./Reference";
+import ColorExample from "./ColorExample";
+import LengthExample from "./LengthExample";
+import FontWeightExample from "./FontWeightExample";
+import FontSizeExample from "./FontSizeExample";
+import FontFamilyExample from "./FontFamilyExample";
+import BorderWidthExample from "./BorderWidthExample";
+import BorderRadiusExample from "./BorderRadiusExample";
+import ShadowExample from "./ShadowExample";
+import SpacingExample from "./SpacingExample";
+
+const SCHEMA_TYPES = {
+  COLOR: "color",
+  LENGTH: "length",
+  BORDER_RADIUS: "borderRadius",
+  BORDER_WIDTH: "borderWidth",
+  FONT_FAMILY: "fontFamily",
+  FONT_SIZE: "fontSize",
+  FONT_WEIGHT: "fontWeight",
+  LINE_HEIGHT: "lineHeight",
+  SHADOW: "shadow",
+  SPACING: "spacing"
+};
+
+function renderTypeExample(props) {
+  switch (props.schema.type) {
```

Comment:
This can be rewritten to be faster and more concise by using a hash table.

```js
const examplesBySchemaType = {
  [SCHEMA_TYPES.COLOR]: ColorExample,
  [SCHEMA_TYPES.LENGTH]: LengthExample
  // etc.
};

function renderTypeExample(props) {
  const Example = examplesBySchemaType[props.schema.type];

  if (!Example) {
    throw new Error(`Unrecognized role type ${props.schema.type}`);
  }

  return <Example {...props} />;
}
```